CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ endif()
 px4_add_module(
 	MODULE lib__ecl
 	STACK_MAIN 6000
-	COMPILE_FLAGS ${MODULE_CFLAGS} -Os
+	COMPILE_FLAGS ${MODULE_CFLAGS}
```

Comment:
Agreed. And while you're at it, line 33-36 can be removed and `STACK_MAX 6000` can be added instead